neuro_data/static_images/data_schemas.py

```diff
@@ -33,7 +33,7 @@
     'stimulus.ColorFrameProjector': ('image_id', 'image_class'),
 }
 
-IMAGE_CLASSES = 'image_class in ("imagenet", "imagenet_v2_gray", "imagenet_v2_rgb")' # all valid natural image classes
+IMAGE_CLASSES = 'image_class in ("imagenet", "imagenet_v2_gray", "imagenet_v2_rgb", "imagenet_v2_rgb_g_b_channels_separately_joined")' # all valid natural image classes
```

Comment:
@Synicix are these all classes or do we need to update this list with Donnie's latest ones?